greenpeace/planet4-master-theme
# PLANET-7532 Retire unused feature flags

- Remove "Purge Cloudflare HTML cache on feature changes".
- Removed now obsolete hooks triggered on saving a feature flag.
- Remove "WordPress template editor".
- Add migration to clean up those options from database.

---

Ref: https://jira.greenpeace.org/browse/PLANET-7532

### Testing

1. Enable both of those features from the Admin Panel
2. Switch to this branch
3. Verify that both options are not visible anymore in the Admin Panel
4. Run the migration manually: `npx wp-env run cli wp p4-run-activator`
5. Verify that the option are no longer in the database: `npx wp-env run cli wp option get planet4_features`
